host/sgx/sgxquoteprovider.c

```diff
@@ -29,11 +29,8 @@ void oe_quote_provider_log(sgx_ql_log_level_t level, const char* message)
 {
     char formatted[OE_LOG_MESSAGE_LEN_MAX] = {0};
     oe_log_level_t oe_log_level[] = {
-        OE_LOG_LEVEL_ERROR,
-        OE_LOG_LEVEL_WARNING,
-        OE_LOG_LEVEL_INFO,
-        OE_LOG_LEVEL_NONE};
-    const char* dcap_level_strings[] = {"ERROR", "WARN", "INFO", "NONE"};
+        OE_LOG_LEVEL_ERROR, OE_LOG_LEVEL_INFO, OE_LOG_LEVEL_NONE};
+    const char* dcap_level_strings[] = {"ERROR", "INFO", "NONE"};
```

Comment:
Does this mean there is no `WARNING` level in the QPL? What about the Azure-Dcap case?

Reply:
Yes, there are only `ERROR` and `INFO` levels in Intel QPL. Az-dcap should follow the public API definitions set by Intel QPL.